setup.py

```diff
@@ -34,6 +36,7 @@ def rtd_dependent_deps():
     "lint": lint_deps,
 }
 
+subprocess.Popen(protoc_cmd)
```

Comment:
We should use `subprocess.checkout_output` for this https://docs.python.org/3/library/subprocess.html#subprocess.check_output